### PR TITLE
Add ScottoMouse Handwired Macropad

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Added handwired ScottoMouse macropad with default and scotto keymaps.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scottomouse/rules.mk [hunk 1]
```diff
@@ -0,0 +1 @@
+BOOTLOADER = rp2040
```
Comment (zvecr) [inline keyboards/handwired/scottokeebs/scottomouse/rules.mk:1]: This is already specified in info.json (and thus produces lint warnings)
```suggestion
# This file intentionally left blank
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/scottokeebs/scottomouse/info.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "layouts": {
+        "LAYOUT_1x6": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0 },
+                { "matrix": [0, 1], "x": 1, "y": 0 },
+                { "matrix": [0, 2], "x": 2, "y": 0 },
+                { "matrix": [0, 3], "x": 3, "y": 0 },
+                { "matrix": [0, 4], "x": 4, "y": 0 },
+                { "matrix": [0, 5], "x": 5, "y": 0 }
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottomouse/info.json:35]:
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scottomouse/readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# ScottoMouse
+
+![ScottoMouse](https://i.imgur.com/yjnhyXWh.jpeg)
+
+A 6 key mouse using [QMK mouse keys](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_mouse_keys.md), case files available [here](https://github.com/joe-scotto/scottokeebs.git).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: Raspberry Pi Pico
+-   Hardware Availability: [ScottoKeebs](https://scottokeebs.com), [Amazon](https://amazon.com), [Ali](https://amazon.com)
```
Comment (filterpaper) [inline keyboards/handwired/scottokeebs/scottomouse/readme.md:9]:
```suggestion
*   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
*   Hardware Supported: Raspberry Pi Pico
*   Hardware Availability: [ScottoKeebs](https://scottokeebs.com), [Amazon](https://amazon.com), [Ali](https://amazon.com)
```

Template consistency suggestion.

File: keyboards/handwired/scottokeebs/scottomouse/info.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0018",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_1x6": {
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottomouse/info.json:28]:
```suggestion
        "LAYOUT": {
```

File: keyboards/handwired/scottokeebs/scottomouse/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_1x6(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottomouse/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scottomouse/keymaps/scotto/keymap.c [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+const uint16_t PROGMEM scroll_up[] = {KC_MS_BTN1, KC_MS_BTN2, KC_MS_UP, COMBO_END};
+const uint16_t PROGMEM middle_click[] = {KC_MS_BTN1, KC_MS_BTN2, COMBO_END};
+combo_t key_combos[] = {
+    COMBO(scroll_down, KC_MS_WH_DOWN),
+    COMBO(scroll_up, KC_MS_WH_UP),
+    COMBO(middle_click, KC_MS_BTN3)
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_1x6(
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottomouse/keymaps/scotto/keymap.c:30]:
```suggestion
    [0] = LAYOUT(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please revert the submodule changes.

**Comment by joe-scotto** [comment]
Done in [746798d](https://github.com/qmk/qmk_firmware/pull/21074/commits/746798d5eae2c81dd69ce08f2d9955c4d367855d)

**Review by fauxpark — APPROVED** [review]

**Pull request merged**